pkg/hs/king/lib/Vere/Serf.hs

```diff
@@ -367,7 +367,7 @@ updateProgressBar count startMsg = \case
       -- We only construct the progress bar on the first time that we
       -- process an event so that we don't display an empty progress
       -- bar when the snapshot is caught up to the log.
-      putStrLn startMsg
+      logTrace $ display startMsg
```

Comment:
Should this be a logTrace? We actually want the text to print right above the progress bar, describing what the progress bar is.

Reply:
Progress bar now uses log system instead of printing. I guess I should add a way to include a description with the progress bar?